cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -255,56 +255,46 @@
+  if ( document.body.parentElement.className.indexOf( 'no-js' ) === -1 ) {
+    {# Include site-wide JavaScript. #}
+    var a = [ '{{ static('js/routes/common.js') }}' ];
     {# Check and include template-level JavaScript. #}
-    function loadScriptTemplate(){
-      {% macro template_js() %}
-         {% include 'js/routes/' + request.path.split('/')[1] + '/single.js' ignore missing %}
-      {% endmacro %}
-      {% set js_source = template_js() | trim %}
-      {% if js_source | length > 0 %}
-        loadScript('{{ static('js/routes/' + request.path.split('/')[1] + '/single.js') }}');
-      {% endif %}
-    }
+    {% macro template_js() %}
+       {% include 'js/routes/' + request.path.split('/')[1] + '/single.js' ignore missing %}
+    {% endmacro %}
+    {% set js_source = template_js() | trim %}
+    {% if js_source | length > 0 %}
+      a.push( '{{ static('js/routes/' + request.path.split('/')[1] + '/single.js') }}' );
+    {% endif %}
+
     {# Check and include component-level JavaScript. #}
-    function loadScriptComponent(){
-      {% if page and page.media %}
-         {% for js in page.media %}
-           loadScript('{{ static('js/routes/on-demand/' + js) }}');
-         {% endfor %}
-      {% endif %}
-    }
+    {% if page and page.media %}
+       {% for js in page.media %}
+         a.push( '{{ static('js/routes/on-demand/' + js) }}' );
+       {% endfor %}
+    {% endif %}
+
     {# TODO: Remove the below block when all pages
              using this template are moved to Wagtail. #}
-    function loadScriptPage(){
-      {# Check and include page-level JavaScript. #}
-      {% macro page_js() %}
-         {% include 'js/routes/' + request.path[1:] + 'index.js' ignore missing %}
-      {% endmacro %}
-      {% set js_source = page_js() | trim %}
-      {% if js_source | length > 0 %}
-        loadScript('{{ static('js/routes/' + request.path[1:] + 'index.js') }}');
-      {% endif %}
-    }
-
-    if (document.body.parentElement.className.indexOf('no-js')===-1){
-      {# Include site-wide JavaScript. #}
-      loadScript('{{ static('js/routes/common.js') }}', function(){
-        loadScriptTemplate();loadScriptComponent();loadScriptPage();
-      });
-    }
-</script>
+    {# Check and include page-level JavaScript. #}
+    {% macro page_js() %}
+       {% include 'js/routes/' + request.path[1:] + 'index.js' ignore missing %}
+    {% endmacro %}
+    {% set js_source = page_js() | trim %}
+    {% if js_source | length > 0 %}
+      a.push( '{{ static('js/routes/' + request.path[1:] + 'index.js') }}' );
+    {% endif %}
 
-<script>
-//<![CDATA[
-    var usasearch_config = { siteHandle: 'cfpb' };
-    loadScript('https://search.usa.gov/javascripts/remote.loader.js');
-//]]>
+    {# Minified dynamic JavaScript loader that injects a script tag in the head
+      of the page. It also provides a callback parameter. #}
```

Comment:
We don't need the callback anymore in this PR, right?

Reply:
I think that's right. The callback is gone, since we're no longer using `loadScript`. I think this means we can also get rid of `cfgov/unprocessed/js/modules/util/js-loader.js`? Looks like `VideoPlayer.js` may still be using it, though.